modules/markdown/markdown_test.go

```diff
@@ -290,8 +290,8 @@ func TestMarkdown(t *testing.T) {
 		Convey("To the internal issue tracker", func() {
 			Convey("It should correctly convert URLs", func() {
 				testCases := []string{
-					"http://localhost:3000/user/project/commit/d8a994ef243349f321568f9e36d5c3f444b99cae", " <code><a href=\"http://localhost:3000/user/project/commit/d8a994ef243349f321568f9e36d5c3f444b99cae\">d8a994ef24</a></code>",
```

Comment:
We can allow 7-char SHA, does not mean we want to show with 7-char in default, please adjust this change.